spec/swap_child_with_successors.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
Obviously this is a hack, wanted to get a test in the PR for review before dealing with submodules.